docs/index.md

```diff
@@ -80,6 +80,53 @@ ansible-playbook playbooks/example-use/playbook.yml -u <managed_node_username> -
 To configure a custom install, you will need to set configuration variables. In the simplest case,
 they can be set in the playbook. See the ansible docs for more flexible idiomatic alternatives.
 
+Note on Plugin Version Compatibility with Pulpcore
+--------------------------------------------------
+
+Pulp 3 has a plugin architecture so that new content types, and new features, can be added by the larger community. However, both pulpcore & plugins are installed via pip, which has limited dependency resolution. Plugins release at their own lifecycles. Thus in the worst case scenario, the 2 stable branches of plugin pulp_juicy could depend on the current branch of pulpcore, while the 2 stable branches of pulp_sugary could depend on an older branch of pulpcore.
```

Comment:
Can this whole PR be line-wrapped please? All pulp docs use a max line width of 100 chars.